tests/ui/traits/new-solver/winnow-specializing-impls.rs

```diff
@@ -0,0 +1,9 @@
+// build-pass
```

Comment:
can you make this a self-contained test using `feature(min_sepcialization)` instead?

I expect us to significantly change the use of specialization in std (and in general) so at this point this test will end up being useless